api.js

```diff
@@ -1902,4 +1902,47 @@ API.prototype.ignoreApdex = function ignoreApdex() {
   transaction.ignoreApdex = true
 }
 
+/**
+ * Runs a function synchronously within a provided LLM custom attributes context and returns its return value.
```

Comment:
I don't think it has to be synchronously.  The context manager should properly bind all functions run within the context

Reply:
I went with how they describe it in the nodejs [docs](https://nodejs.org/api/async_context.html#asynclocalstoragerunstore-callback-args). 

```
Runs a function synchronously within a context and returns its return value. The store is not accessible outside of the callback function. The store is accessible to any asynchronous operations created within the callback.
```

Reply:
>  The store is accessible to any asynchronous operations created within the callback.